File: Back-end/Back-end/Entidades/Usuario.cs
using Back_end.Validaciones;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Back_end.Entidades
{
    public class Usuario
    {
        public int id { get; set; }

        public string nombre { get; set; }
        public string apellido { get; set; }
        public string correo_electronico { get; set; }
        public DateTime fecha_nacimiento { get; set; }
        public int telefono { get; set; }
        public string pais { get; set; }
        public bool contactar { get; set; }

        
    }
}

File: Front-end/front-end/src/app/usuarios/usuario.ts
export interface usuarioCreacionDTO{
    nombre: string;
    apellido: string;
    correo_electronico: string;
    fecha_nacimiento: Date;
    telefono: string;
    pais: string;
    contactar: boolean;
}

export interface usuarioDTO{
    id: number;
    nombre: string;
    apellido: string;
    correo_electronico: string;
    fecha_nacimiento: Date;
    telefono: string;
    pais: string;
    contactar: boolean;
}